app/game/index.ts

```diff
@@ -3,10 +3,13 @@ import { put } from "@vercel/blob"
 import { kv } from "@vercel/kv";
 import { randomUUID } from "crypto";
 
+export type Variant = "thavalon" | "avalon" | "jealousy";
```

Comment:
Is "avalon" for a future implementation? Might make more sense to leave out until implemented.

Reply:
valid